app/views/faqs/_faq.html.erb

```diff
@@ -0,0 +1,15 @@
+<div class="card faq-item mb-3" id="<%= dom_id faq %>">
+  <div class="card-header bg-primary text-white fw-bold d-flex gap-2">
+    <span class="fw-bold">Question:</span>
+    <span><%= faq.question %></span>
+  </div>
+  <div class="card-body bg-light d-flex gap-2">
+    <p class="card-text">Answer: <%= faq.answer %></p>
+  </div>
+  <% if is_super_admin? %>
```

Comment:
I think edit and delete should be available to admins, not super_admins. Please replace is_super_admin? with is_admin?